website/README.md

```diff
@@ -47,6 +47,7 @@ Use Jekyll to serve the website locally (by default, at
 `http://localhost:8080`):
 
 ```sh
+make js
```

Comment:
```suggestion
$ make js
```

Reply:
Out of curiosity: Why it's common practice to add $ before each line in shell examples? It just makes copy & pasting harder :D

Reply:
it's because most terminal prompt strings end in "$". the $ generally indicates a normal user where # is used to indicate root, for a quick visual indication of whether or not you are root. the $ in the lines of the example are an indication that these are lines to be run in the terminal, otherwise it might be ambiguous what the purpose of these example lines is. in this case, if you're familiar with `make` and `cd` then it's probably pretty clear but if you saw something like:

```
find '.'
```

then it might be significantly more ambiguous depending on the context that you should run this line at your terminal.

also [here's](https://superuser.com/questions/57575/what-is-the-origin-of-the-unix-dollar-prompt) some quick history of $, apparently it dates at least as far back as unix 7.